Rakefile

```diff
@@ -71,6 +75,63 @@ def re_run_basic_smoke_test
   sh beaker
 end
 
+def get_passing_json(current_branch_trunk)
```

Comment:
I have seen the word "trunk" used in SVN to denote what would be the equivalent of "master" for us. I might have been using trunk wrong in the past but I think that "current_branch" would provide the similar level of understanding in the code here.